apps/blog/_posts/tech-post-1.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "[Tech Post #1] Como nasceu a plataforma do Coderdojo Braga?"
+date: "2022-09-17"
+author:
+  name: "Rui Oliveira"
+  photo: "ruioliveira02.jpg"
+  username: "ruioliveira02"
+draft: false
+topic: "Desenvolvimento"
+featured: true
+---
+
+---
+
+&nbsp;
+
+Este é o primeiro post de uma série em que vamos discutir as plataformas do Coderdojo Braga. Como foram feitas, dificuldades técnicas, histórias engraçadas e para onde vão são alguns dos temas que podem esperar daqui. Espero sinceramente que, antes de um blog técnico, que isto seja um blog educativo, e que demonstre o trabalho necessário para fazer uma aplicação que vai ser usada por várias pessoas, e traga uma maior sensibilidade para a engenharia de software.
+
+&nbsp;
+
+## O nascimento
+
+Podem só ter sido lançadas recentemente, mas esta plataforma já se encontra em desenvolvimento há mais de um ano e meio. Se formos a ver o histórico no [Github](https://github.com/coderdojobraga/bokken), vemos que o desenvolvimento começou em março de 2021.
```

Comment:
You should use only singular or only plural. There should be coherence.

Also, wouldn't it be better to link to the initial commits of each repo?
- Bokken: https://github.com/coderdojobraga/bokken/commit/5729ea8239adcbafa1e9b2a9e6df6fe779e7908a
- Shuriken: https://github.com/coderdojobraga/shuriken/commit/1522c8edfa9ccf320dd0e4a02f570f44524d46b0

Reply:
I will only link 1, as I don't see a neat way to link to both at the same time (plus, those commits happened in the same day)